### DISCUSSION TITLE
⛔ Show post content

### DISCUSSION THREAD

**Discussion by Systemedic** [elementor/elementor#15195] [Feature Request]
Hi,

It would be very nice to have the option to show the post content using the post block. The content should be shown including (working) links and text make-up. This way a end-user can simply edit their text by going in to the post section, or a custom! post type without having to open the editor.

This way end user can simply us a familiar interface to word (classic editor) to edit their texts.

**Comment by nicholaszein** [comment]
Hello @Systemedic,

Thank you for submitting a feature request! 🙏 

### ⛔ 🚀 After reviewing your request, we have decided not to add this to our roadmap for the time being.

Please feel free to contact us again with other feature suggestions. 😉 

Best regards
